openexp/_canvas/psycho.py

```diff
@@ -45,6 +45,18 @@
 	raise osexception(
 		u'PsychoPy is missing the GratingStim() class. Please update your version of PsychoPy! For installation instructions, please visit http://www.psychopy.org/.')
 
+import psychopy, warnings
+from distutils.version import StrictVersion
+try:
+    PsyPy_vers = StrictVersion(psychopy.__version__)
+except ValueError:
+    warnings.warn(u'Invalid version number %s of PsychoPy'%psychopy.__version__)
+else:
+	if PsyPy_vers.version < (1, 87):
+		warnings.warn(u'Your PsychoPy version is too old. You will not be able\
```

Comment:
It's a matter of style, but I prefer implicit string concatenation over these forced line breaks. In Python `'x' 'y'` becomes `'xy'`. And this works across lines too:

~~~ .python
		warnings.warn(u'Your PsychoPy version is outdated. To draw complex '
			u'shapes that include holes, concavities and such, please update '
			u'PsychoPy to version 1.84.1 or higher.')
~~~